tc/sgx/trusted_worker_manager/enclave/CMakeVariables.txt

```diff
@@ -100,16 +110,16 @@ FOREACH(lib ${SGX_LIBS_UNTRUSTED_NAMES})
 ENDFOREACH(lib)
 
 # XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-FUNCTION(SGX_EDGE_TRUSTED EDL EDGE_FILES)
+FUNCTION(SGX_EDGE_TRUSTED EDL EDL_PATH EDGE_FILES)
     GET_FILENAME_COMPONENT(EDL_BASE_NAME ${EDL} NAME_WE)
     GET_FILENAME_COMPONENT(EDL_DIR_NAME ${EDL} DIRECTORY)
     INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
 
     SET (EDGE_FILES_LIST "${CMAKE_CURRENT_BINARY_DIR}/${EDL_BASE_NAME}_t.h" "${CMAKE_CURRENT_BINARY_DIR}/${EDL_BASE_NAME}_t.c")
     SET (${EDGE_FILES} ${EDGE_FILES_LIST} PARENT_SCOPE)
     ADD_CUSTOM_COMMAND( OUTPUT ${EDGE_FILES_LIST}
-	    COMMAND "${SGX_EDGER}" --trusted ${EDL} --search-path ${SINGLETON_PATH}
-	    --search-path ${KME_PATH} --search-path ${WPE_PATH} --search-path ${SGX_INCLUDE}
+	    COMMAND "${SGX_EDGER}" --trusted ${EDL} --search-path ${EDL_PATH}/..
```

Comment:
Can we pass the required `EDL_PATH` instead of `${EDL_PATH}/..`

Reply:
Done